osu.Game.Rulesets.Osu/Difficulty/OsuDifficultyAttributes.cs

```diff
@@ -52,6 +52,14 @@ public class OsuDifficultyAttributes : DifficultyAttributes
         [JsonProperty("speed_difficult_strain_count")]
         public double SpeedDifficultStrainCount { get; set; }
 
+        // DEV ATTRIBUTE - DO NOT STORE
+        [JsonProperty("aim_relevant_object_count")]
+        public double AimRelevantObjectCount { get; set; }
+
+        // DEV ATTRIBUTE - DO NOTE STORE
+        [JsonProperty("speed_relevant_object_count")]
+        public double SpeedRelevantObjectCount { get; set; }
+
```

Comment:
I know they're commented as "dev" attributes but please get rid of these.

Reply:
removed